stats.py

```diff
@@ -690,6 +717,16 @@ def handle_options(options):
 
     pd.set_option("display.max_rows", len(df))
     print(df)
+
+    count_ws = list(df["Correct"]).count(False)
```

Comment:
Pandas is awesome, isn't it?

Reply:
@lubien see the power

Reply:
![](https://media1.tenor.com/images/34effa9c87833d193ae5f0ccdb2d12b3/tenor.gif)

Reply:
🐼  Hahaaha